providers/terraform-provider-csbsqlserver/csbsqlserver/provider_test.go

```diff
@@ -0,0 +1,96 @@
+package csbsqlserver_test
+
+import (
+	"csbbrokerpakazure/providers/terraform-provider-csbsqlserver/csbsqlserver"
+	"encoding/json"
+	"fmt"
+	"regexp"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Provider Configuration", func() {
+	var (
+		server           string
+		port             int
+		providerUsername string
+		providerPassword string
+		database         string
+		encrypt          string
+		bindingUsername  string
+		bindingPassword  string
+		bindingRoles     []string
+	)
+
+	BeforeEach(func() {
+		server = "flopsy.com"
+		port = 1234
+		providerUsername = "mopsy"
+		providerPassword = "cottontail"
+		database = "peter"
+		encrypt = ""
+		bindingUsername = "username"
+		bindingPassword = "password"
+		bindingRoles = make([]string, 0) // JSON of empty slice differs from nil slice
+	})
+
+	DescribeTable(
+		"validation of parameters",
+		func(cb func(), expectError string) {
+			cb()
```

Comment:
Slightly unclear as to what `cb` stands for here?

Reply:
It's now been renamed to "callback"